src/go/rdctl/pkg/autostart/autostart_darwin.go

```diff
@@ -0,0 +1,9 @@
+package autostart
+
+import (
+	"errors"
+)
+
+func EnsureAutostart(autostartDesired bool) error {
+	return errors.New("not implemented")
```

Comment:
After testing this, I would prefer a more informative message, like `"setup --auto-start not implemented on macOS"`